Makefile

```diff
@@ -169,10 +169,14 @@ cfn/lint: | guard/program/cfn-lint
 
 ## Runs eclint against the project
 eclint/lint: | guard/program/eclint guard/program/git
-eclint/lint: ECLINT_PREFIX ?= git ls-files -z | xargs -0
+eclint/lint: HAS_UNTRACKED_CHANGES ?= $(shell git diff-index --quiet HEAD -- || echo "true")
+eclint/lint: ECLINT_PREFIX ?= git ls-files -z
 eclint/lint:
 	@ echo "[$@]: Running eclint..."
-	cd $(PROJECT_ROOT) && $(ECLINT_PREFIX) eclint check
+	cd $(PROJECT_ROOT) && \
```

Comment:
Why do we need to `cd` again? Shouldn't we be in the project root already when this executes? The `cd` is complicating the make target, forcing the `&&` chaining...

Reply:
its a quirk of how the env is built. I have to specify PROJECT_ROOT b/c the bindmount for the project name is project specific. I suppose I could make things more generic but that might make log outputs generated in docker a little harder to track relative to the target project(s)